app/presenters/project_presenter.rb

```diff
@@ -22,13 +25,12 @@ def severity
   end
 
   def git_remote
-    stage = ordered_stages&.detect{ |s| s.name == "master" }
-    stage && stage.git_remote
+    stage = ordered_stages&.detect { |s| s.name == 'master' }
+    stage&.git_remote
   end
 
-  def is_kubernetes?
-    hokusai_stages = ordered_stages&.detect{ |s| s.hokusai }
-    !hokusai_stages.nil?
+  def is_kubernetes? # rubocop:disable Naming/PredicateName
+    true if ordered_stages&.detect { |s| !s.hokusai&.empty? }
```

Comment:
```suggestion
    ordered_stages&.any? { |s| !s.hokusai&.empty? }
```

Not sure if returning `nil` is specifically important, but the `#any` method could make it simpler a bit.